cleanlab/internal/multilabel_rank_utils.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (C) 2017-2022  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+Helper classes and functions used internally for computing multilabel label quality scores.
```

Comment:
```suggestion
Helper classes and functions used internally to compute label quality scores in multi-label  classification.
```

Reply:
Let's try to stick with naming convention "multi-label  classification" throughout. @aditya1503 

Just referring to it as "multi-label" has potential confusion with multi-annotator (as well as other settings we will introduce in the future)